### DISCUSSION TITLE
ios tv? spoof for youtube?

### DISCUSSION THREAD

**Discussion by yonggamer** [ReVanced/revanced-discussions#1126] [Question] [answered]
i can not find a iOS TV on google
![ios](https://github.com/user-attachments/assets/17ce6933-ad11-4660-95f8-83b7a0258a04)
i want a explaination for this

**Comment by oSumAtrIX** [comment]
https://tv.apple.com/

**Reply by yonggamer** [reply]
that is apple tv not iOS TV

**Reply by oSumAtrIX** [reply]
iOS TV IS for Apple TV

**Reply by yonggamer** [reply]
i see this https://en.wikipedia.org/wiki/TvOS
but no iOS TV

**Reply by oSumAtrIX** [reply, marked as answer]
iOS TV is TvOS, I can repeat this a couple more times, but I don't see the point. If you wanna proof it yourself, find the Enum type in the compiled YouTube code. You will see IOS_TV.
